src/constants.ts

```diff
@@ -20,3 +20,7 @@ export const getRequestTimeout = 120 * 1000; // ms
 export const getRequestPollInterval = 2 * 1000; // ms
 // UTM
 export const registrationURL = "https://app.snyk.io/signup/?utm_medium=Partner&utm_source=RedHat&utm_campaign=Code-Ready-Analytics-2020&utm_content=Register";
+
+//telemetry event names
+export const reportOpenedNotificationPopup = 'Notification pop-up - Open report Clicked'
```

Comment:
Define separate module for telemetry, or make it part of redhatTelemetry module

Reply:
PTAL